src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -293,7 +293,8 @@ export class NativeStakingMapper {
 
     const estimatedCompletionTime =
       transaction.executionDate.getTime() +
-      networkStats.estimated_exit_time_seconds * 1000;
+      networkStats.estimated_exit_time_seconds * 1000 +
+      networkStats.estimated_withdrawal_time_seconds * 1000;
```

Comment:
Nit: I'd also update the function doc from:
```
...
- If the execution date is available, the status is {@link StakingValidatorsExitStatus.RequestPending} if the current date is before the estimated exit time, otherwise the status is {@link StakingValidatorsExitStatus.ReadyToWithdraw}.
   ...
```

To: 

```
...
- If the execution date is available, the status is {@link StakingValidatorsExitStatus.RequestPending} if the current date is before the estimated exit time + the estimated withdrawal time, otherwise the status is {@link StakingValidatorsExitStatus.ReadyToWithdraw}.
   ...
```

(Or something similar)

Reply:
Updated in 321c281cb0332198bdc8ca1c4dde81ee90c7be71.